airflow/models/baseoperator.py

```diff
@@ -1824,11 +1751,55 @@ def get_serialized_fields(cls):
                     'operator_extra_links',
                     'upstream_task_ids',
                     'task_type',
+                    # These are automatically populated from partial_kwargs. In
+                    # a perfect world, they should be properties like other
+                    # partial_kwargs-populated values e.g. 'queue' below, but we
+                    # must match BaseOperator's implementation and declare them
+                    # as writable attributes instead.
+                    'weight_rule',
+                    'priority_weight',
+                    'trigger_rule',
```

Comment:
```suggestion
                    'trigger_rule',
                    'depends_on_past',
                    'wait_for_downstream',
```

Reply:
Oh plus all the other ones you added

Reply:
Those are calculated properties and shouldn’t be included in `attr.fields_dict()` in the first place. Or do you want them to be included anyway just to be safe?

Reply:
Oh, if they aren't that's fine.